sdk/search/ai-search/README.md

```diff
@@ -0,0 +1,302 @@
+# Azure Cognitive Search client library for JavaScript
+
+[Azure Cognitive Search](https://docs.microsoft.com/en-us/azure/search/) is a search-as-a-service cloud solution that gives developers APIs and tools for adding a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.
+
+Azure Cognitive Search is well suited for the following application scenarios:
+
+- Consolidation of heterogeneous content types into a private, single, searchable index. Queries are always over an index that you create and load with documents, and the index always resides in the cloud on your Azure Cognitive Search service. You can populate an index with streams of JSON documents from any source or platform. Alternatively, for content sourced on Azure, you can use an indexer to pull data into an index. Index definition and management/ownership is a key reason for using Azure Cognitive Search.
+- Raw content is large undifferentiated text, image files, or application files such as Office content types on an Azure data source such as Azure Blob storage or Cosmos DB. You can apply cognitive skills during indexing to add structure or extract meaning from image and application files.
+- Easy implementation of search-related features. Azure Cognitive Search APIs simplify query construction, faceted navigation, filters (including geo-spatial search), synonym mapping, typeahead queries, and relevance tuning. Using built-in features, you can satisfy end-user expectations for a search experience similar to commercial web search engines.
+- Indexing unstructured text, or extracting text and information from image files. The AI enrichment feature of Azure Cognitive Search adds AI processing to an indexing pipeline. Some common use-cases include OCR over scanned document, entity recognition and key phrase extraction over large documents, language detection and text translation, and sentiment analysis.
+- Linguistic requirements satisfied using the custom and language analyzers of Azure Cognitive Search. If you have non-English content, Azure Cognitive Search supports both Lucene analyzers and Microsoft's natural language processors. You can also configure analyzers to achieve specialized processing of raw content, such as filtering out diacritics.
+
+Use the client library to:
+
+- Create and manage search indexes.
+- Upload and update documents in the search index.
+- Manage indexers that pull data from a data source into an index.
+- Query documents in the index with a powerful set of search APIs that support faceted navigation, typeahead queries, suggestions, and geo-spatial search.
+- Enrich your search index with AI skills that add structure or extract meaning from raw documents during indexing.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/search/ai-search/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/ai-search) |
+[API reference documentation](https://aka.ms/azsdk-js-search-ref-docs) |
```

Comment:
I am grateful for the URL shortener here but I don't think I've ever noticed how difficult it would be to memorize that... :)

Reply:
it's an aka.ms link not so you can memorize it but more so we can point it at MSDN after it gets published there. I followed the same convention as text analytics